packages/cli/src/config.ts

```diff
@@ -3,7 +3,7 @@ import { resolve } from 'path';
 import type { GenerateOptions } from './generate';
 import type { IntrospectionOptions } from './introspection';
 
-export type GqlessConfig = GenerateOptions & {
+export type GQlessConfig = Omit<GenerateOptions, 'endpoint'> & {
```

Comment:
This type name change is a breaking change and the previous version should be also provided with the `@deprecated` comment